pkgs/development/libraries/science/biology/bicgl/default.nix

```
@@ -26,6 +28,6 @@ stdenv.mkDerivation rec {
     description = "Brain Imaging Centre graphics library";
     maintainers = with maintainers; [ bcdarwin ];
     platforms = platforms.unix;
-    license   = licenses.free;
+    license = licenses.free;
```

Could not find license in the repo https://github.com/BIC-MNI/bicgl
```suggestion
    license = licenses.unfree;
```

---

This project appears to be part of an umbrella project with a GPL-3.0 license: 

https://github.com/BIC-MNI/minc-toolkit-v2

---

Fixed upstream.